persistence/src/main/java/org/mskcc/cmo/metadb/persistence/SampleMetadataRepository.java

```diff
@@ -35,5 +38,5 @@
         + "MERGE (pm)-[:PX_TO_SP]->(sm)"
         + "RETURN sm"
     )
-    void saveSampleMetadata(@Param("sample") SampleMetadataEntity sample);
+    void saveSampleMetadata(@Param("sample") SampleMetadataEntity sample); 
```

Comment:
For the purposes of concurrency, we will need to have this method return the SampleMetadataEntity that was just persisted (including the uuid that was generated).